src/style.css

```diff
@@ -2,2 +1,56 @@
+html {
+  font-family: Roboto, serif;
+  font-size: 12px;
```

Comment:
The base font size is set to 12px, which might be too small for readability on some devices. Consider using a relative unit like 'em' or 'rem' for better scalability and accessibility.